### DISCUSSION TITLE
Use tape with WebDriverIO?

### DISCUSSION THREAD

**Discussion by bhootd** [webdriverio/webdriverio#10169] [Q&A] [answered]
`tape` is a simple test framework which also provides a small set of assertions.

I wonder if we could use WebDriverIO to drive the test and tape to initialise and assert a test.

Prior art seems to exist:

- https://gist.github.com/patrickarlt/931b129598703eed4dda
- https://github.com/dfranklinau/webdriverio-tape

**Comment by christian-bromann** [comment, marked as answer]
> I wonder if we could use WebDriverIO to drive the test and tape to initialise and assert a test.

Yes, you can use Tape with WebdriverIO. However there is no support for it when using the WDIO testrunner. This means you can not just use all the reporter and services provided by the ecosystem. It is possible to wire this together manually though.
